R/reportEmployment.R

```diff
@@ -11,15 +10,16 @@
 #' @author Aman Malik
 #' @examples
 #' \dontrun{
-#' reportEmployment(gdx, improvements = "All", multiplier = "own", subtype = "expert", 
-#' share_manf = "local", decline = "capcosts",type="per_year")
+
+#' reportEmployment(gdx, improvements = "All", multiplier = "own", subtype = "expert", share_manf = "local", decline = "capcosts")
 #' }
 #' @importFrom madrat calcOutput
 #' @importFrom magclass getNames add_columns getItems
 #' @export
 
-reportEmployment <- function(gdx, improvements, multiplier, subtype, share_manf, decline, type) {
+reportEmployment <- function(gdx, improvements, multiplier, subtype, share_manf, decline) {
 
+  # use setConfig(forcecache=T) to make the code run faster
```

Comment:
This seems like a dangerous hint as it is. I suggest either explaining a bit further in which situation forcecache might be helpful and what problems might arise from it, or to delete this comment and leave explaining forcecache to the madrat package.